setup_dev_env.sh

```diff
@@ -132,4 +137,4 @@ if [ "$NP4_INTEL" == YES ]; then
     DOCKER_RUN_OPTIONS="$DOCKER_RUN_OPTIONS -v /dev/intel-fpga-fme.0:/dev/intel-fpga-fme.0"
 fi
 DOCKER_RUN_OPTIONS="$DOCKER_RUN_OPTIONS $@"
-docker run $DOCKER_RUN_OPTIONS -w /stratum --user $USER -ti $IMAGE_NAME bash
+docker run $DOCKER_RUN_OPTIONS -w /stratum --user $DOCKER_USER -ti $IMAGE_NAME bash
```

Comment:
I just realized that all run options should be contained in the `DOCKER_RUN_OPTIONS` var.

I'll make a separate PR for this.